parsl/monitoring/radios/zmq_router.py

```diff
@@ -108,17 +119,17 @@ def start(self) -> None:
 def zmq_router_starter(*,
                        comm_q: mpq.Queue,
                        resource_msgs: mpq.Queue,
-                       exit_event: Event,
+                       exit_event: EventType,
 
-                       hub_address: str,
-                       zmq_port_range: Tuple[int, int],
+                       address: str,
+                       port_range: Tuple[int, int],
 
                        run_dir: str,
                        logging_level: int) -> None:
     setproctitle("parsl: monitoring zmq router")
     try:
```

Comment:
Is there any debugging or human utility in including some (terse) details in the process title?  Like here:
```python
setproctitle("parsl: monitoring zmq router (booting up)")
```
and then below, after acquiring the port:
```python
setproctitle(f"parsl: monitoring zmq router (port: {router.zmq_receiver_port})")
```
Probably TMI, but thinking generally in terms of visibility and trying to think what a user or administrator might care about as easy reference.

Reply:
not in this PR. maybe interesting to include more info here - a runid/workflow ID would be another thing, for example, to help people associate different processes together in `ps`